src/main/java/core/basesyntax/FigureTwo.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class FigureTwo extends Figure {
+
+    private double firstSide; // длина
+    private double secondSide; // ширина
+
+    public FigureTwo(String color, double firsSide, double secondSide) {
```

Comment:
There is a typo in the constructor parameter name `firsSide`. It should be `firstSide` to match the class property name.